pubspec.yaml

```diff
@@ -1,11 +1,10 @@
 name: another_authenticator
-description: Another two-factor authentication app.
-publish_to: none
-
+description: "Another two-factor authentication app."
+publish_to: 'none'
```

Comment:
Inconsistent quotes. 
But looks like this is consistent with template. 
~~Was this automatic, or manually edited?~~